doc/quickquide.html

```diff
@@ -0,0 +1,558 @@
+
```

Comment:
is still already part of the rendered documentation? Do you mind converting this via pandoc to markdown as it is easier to read in an editor? If the formatting will be screwed, we can just keep the html.

Reply:
I just did a quick hack to make the docs in py.  I didn't have access to the internet so I couldn't look up options.  I will try to see if pandoc can be created instead of html.

Reply:
I looked a bit at converting this with pandoc.  I haven't used it before.  It may be possible though markdown does not have great table support.  I will give it a try when I have a good connection.